docs/advanced-guides/cloud-in-a-box.md

```diff
@@ -45,8 +45,14 @@ There are two types of Cloud in a Box.
 
 ## Installation
 
-1. Download one of the Cloud in a Box images of type sandbox
+:::warning
+The images currently download and install the 
+[latest release of the installation-scripts](https://github.com/osism/cloud-in-a-box),
+therefore it is mandatory to update the installation media when the base release changes.
```

Comment:
This is not necessary. You only have to change the ISO image when we change the underlaying distribution (Ubuntu 22.04 -> Ubuntu 24.04). The ISO images does not have to be changed when there is a new OSISM release.

Reply:
Removed the warning